src/test/scala/spark/dicom/TestDicomDataSource.scala

```diff
@@ -92,6 +92,29 @@ class TestDicomDataSource
       )
     }
 
+    it("reads all attributes of DICOM files") {
+      val df = spark.read
+        .format("dicomFile")
+        .load(SOME_DICOM_FILEPATH)
+        .select("*")
+
+      val row = df.first
+    }
```

Comment:
Shouldn't there be some assert statement in this test as well?

Reply:
This test is successful if no exception is raised.
Dataframe evaluation is forced by calling `.first`.